src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -214,12 +276,26 @@ class TypeLookupManager
         return participant_;
     }
 
-    //! Get out instanceName as defined in 7.6.3.3.4 the XTypes 1.3 document
-    std::string get_instanceName() const;
+    /**
+     * Create instance name as defined in 7.6.3.3.4 the XTypes 1.3 document
```

Comment:
```suggestion
     * Create instance name as defined in section 7.6.3.3.4 XTypes 1.3 specification

Reply:
Updated in [cfa3f78](https://github.com/eProsima/Fast-DDS/pull/4177/commits/cfa3f781c09d678528b673acc9feef170c546bc0)